src/custodian/vasp/jobs.py

```diff
@@ -114,10 +116,19 @@ def __init__(
             auto_gamma (bool): Whether to automatically check if run is a
                 Gamma 1x1x1 run, and whether a Gamma optimized version of
                 VASP exists with ".gamma" appended to the name of the VASP
-                executable (typical setup in many systems). If so, run the
+                executable or with "gamma" replacing "std" in the name of
+                the VASP executable (typical setup in many systems). If so, run the
```

Comment:
@mkhorton: I think this is very smart to do, seeing as VASP ships it this way (and not with `.gamma`). I don't see the change made to the source code though, so if that's the case, just a head's up!